cmd/get/workflow.go

```diff
@@ -146,32 +157,34 @@ func getWorkflowFunc(ctx context.Context, args []string, cmdCtx cmdCore.CommandC
 
 	logger.Debugf(ctx, "Retrieved %v workflows", len(workflows))
 	if config.GetConfig().MustOutputFormat() == printer.OutputFormatTABLE {
-		return adminPrinter.Print(config.GetConfig().MustOutputFormat(), workflowColumns, WorkflowToTableProtoMessages(workflows)...)
+		return adminPrinter.Print(config.GetConfig().MustOutputFormat(), listWorkflowColumns, WorkflowToTableProtoMessages(workflows)...)
 	}
-	return adminPrinter.Print(config.GetConfig().MustOutputFormat(), workflowColumns, WorkflowToProtoMessages(workflows)...)
+	return adminPrinter.Print(config.GetConfig().MustOutputFormat(), listWorkflowColumns, WorkflowToProtoMessages(workflows)...)
 }
 
 // FetchWorkflowForName fetches the workflow give it name.
 func FetchWorkflowForName(ctx context.Context, fetcher ext.AdminFetcherExtInterface, name, project,
-	domain string) ([]*admin.Workflow, error) {
+	domain string) ([]*admin.Workflow, bool, error) {
 	var workflows []*admin.Workflow
```

Comment:
nit: we can use named return variables here, which will initialize them with their zero values so we don't have to do so manually: https://yourbasic.org/golang/named-return-values-parameters/

Reply:
Fixed